src/com/epam/javatrainings/Validator.java

```diff
@@ -0,0 +1,76 @@
+package com.epam.javatrainings;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.regex.*;
+import java.time.Period;
+
+public class Validator <T> {
+    public static final Pattern VALID_EMAIL_ADDRESS_REGEX =  Pattern.compile("^[A-Z0-9._%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,6}$", Pattern.CASE_INSENSITIVE);
+    private ArrayList<Error> errorList;
+    private T dto;
+
+    public Validator(T dto){
+        this.dto = dto;
+        errorList = new ArrayList<Error>();
+    }
+
+    public ArrayList <Error> Validate() {
```

Comment:
Method name should start with a small letter.
Try to modify the method so it takes any object and validates it:
public <T> ArrayList <Error> validate(T dto) {}